src/palace/manager/core/opds2_import.py

```diff
@@ -396,53 +321,48 @@ def _extract_contributors(
 
         for contributor in contributors:
             self.log.debug(
-                "Started extracting contributor metadata from {}".format(
-                    encode(contributor)
-                )
+                f"Started extracting contributor metadata from {contributor.model_dump_json()}"
             )
 
-            if not contributor.name:
-                self.log.warning(
-                    f"Contributor has no name. Skipping. {encode(contributor)}"
-                )
-                continue
+            if isinstance(contributor, rwpm.ContributorWithRole):
+                roles = self._extract_contributor_roles(contributor.roles, default_role)
+            else:
+                roles = [default_role]
 
             contributor_metadata = ContributorData(
                 sort_name=contributor.sort_as,
-                display_name=contributor.name,
+                display_name=str(contributor.name),
```

Comment:
Hmmm. Maybe just food for _future_ thought here, but this makes me realize that, at least for some properties (like contributors), it might not make sense for `name` to be a `LanguageMap`. But maybe it does (e.g., in the case of corporate authors, which might need a translation).

I note also that [the schema](https://readium.org/webpub-manifest/schema/contributor-object.schema.json) calls for a string or a language map.